telegramium-high/src/main/scala/telegramium/bots/high/WebhookBot.scala

```diff
@@ -3,16 +3,17 @@ package telegramium.bots.high
 import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Resource, Sync, Timer}
 import cats.syntax.apply._
 import cats.syntax.flatMap._
+import cats.syntax.foldable._
 import cats.syntax.functor._
 import org.http4s.circe.{jsonOf, _}
 import org.http4s.dsl.Http4sDsl
 import org.http4s.dsl.impl.Path
-import org.http4s.implicits._
+import org.http4s.syntax.kleisli._
 import org.http4s.server.Server
 import org.http4s.server.blaze.BlazeServerBuilder
-import org.http4s.{EntityDecoder, HttpApp, HttpRoutes}
+import org.http4s.{EntityDecoder, HttpRoutes}
 import telegramium.bots.CirceImplicits._
-import telegramium.bots.client.{Method, Methods}
+import telegramium.bots.client.{Method, Methods => Methods1}
```

Comment:
Let's use more meaningful aliases

Reply:
Sure, what do you suggest? `ClientMethods`? I wanted to change this to resolve the warning.

Reply:
ApiMethods, for example